fastlane/lib/fastlane/actions/s3.rb

```diff
@@ -191,7 +196,14 @@ def self.run(config)
         Actions.lane_context[SharedValues::S3_VERSION_OUTPUT_PATH] = version_url
         ENV[SharedValues::S3_VERSION_OUTPUT_PATH.to_s] = version_url
 
+        Actions.lane_context[SharedValues::S3_ICON_OUTPUT_PATH] = icon_url
+        ENV[SharedValues::S3_ICON_OUTPUT_PATH.to_s] = icon_url
+
         UI.success("Successfully uploaded ipa file to '#{Actions.lane_context[SharedValues::S3_IPA_OUTPUT_PATH]}'")
+        UI.success("Successfully uploaded plist file to '#{Actions.lane_context[SharedValues::S3_PLIST_OUTPUT_PATH]}'")
+        UI.success("Successfully uploaded html file to '#{Actions.lane_context[SharedValues::S3_HTML_OUTPUT_PATH]}'")
+        UI.success("Successfully uploaded version file to '#{Actions.lane_context[SharedValues::S3_VERSION_OUTPUT_PATH]}'")
+        UI.success("Successfully uploaded icon file to '#{Actions.lane_context[SharedValues::S3_ICON_OUTPUT_PATH]}'")
```

Comment:
I am not sure about this, I agree it's a little repetitive, but is explicit better in this case?